README.md

```diff
@@ -117,19 +139,23 @@ from https://github.com/facebook/watchman
 ## Requirements
 
 * [Java 8 JDK](http://www.oracle.com/technetwork/java/javase/downloads/index.html)
-* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE)
+* `java` and `jar` on the PATH (make sure you use `java` executable from JDK but not JRE). 
+  * To test this, try running the command `javac`. This command won't exist if you only have the JRE
+  installed. If you're met with a list of command-line options, you're referencing the JDK properly.
+  * If you're running Java 11, you won't be able to build Selenium. We recommend using a tool like  
+  `jenv` to manage different using versions of Java 
 * [Python 2.7](https://www.python.org/)
 * `python` on the PATH (make sure it's Python 2.7, as buck build tool is not Python 3 compatible)
 * [The Requests Library](http://python-requests.org) for Python: `pip install requests`
-* MacOS users should have XCode installed
+* MacOS users should have the latest version of XCode installed, including the command-line tools
 
 Although the build system is based on rake, it's **strongly advised**
 to rely on the version of JRuby in `third_party/` that is invoked by
 `go`.  The only developer type who would want to deviate from this is
 the “build maintainer” who's experimenting with a JRuby upgrade.
 
-Note that all Selenium Java artifacts are **built with Java 8
-(mandatory)**.  Those _will work with any Java >= 8_.
+Note that **all** Selenium Java artifacts are built with **Java 8
+(mandatory)**. Those _will work with any Java >= 8_.
```

Comment:
It's best to install the most recently available version of OpenJDK 8

Reply:
@mmerrell was this comment addressed?

Reply:
OpenJDK 11, now.